tls/s2n_security_policies.c

```diff
@@ -23,144 +23,168 @@ const struct s2n_security_policy security_policy_20170210 = {
     .minimum_protocol_version = S2N_TLS10,
     .cipher_preferences = &cipher_preferences_20170210,
     .kem_preferences = &kem_preferences_null,
+    .signature_preferences = &s2n_signature_preferences_20140601,
 };
 
 const struct s2n_security_policy security_policy_20190801 = {
     .minimum_protocol_version = S2N_TLS10,
     .cipher_preferences = &cipher_preferences_20190801,
     .kem_preferences = &kem_preferences_null,
+    .signature_preferences = &s2n_signature_preferences_20200207,
```

Comment:
It seems a little strange to have 2020 signature preferences on a 2019 security policy. Should we name this differently?

Reply:
I can change it to `s2n_signature_preferences_20190801` to match.